### PR TITLE
update(Developers.md)

### PR DESCRIPTION
**Pull request by mathieuLivebardon**
Add new section in Developers.md: Develop modified ud-viz library in your demo

### CODE REVIEW

File: docs/static/Developers.md [hunk 1]
```diff
@@ -75,6 +75,57 @@ npm run dev-examples
...
+  Entries of the form
+  ```json
+  "dependencies": {
+     "@ud-viz/*": "x.x.x"
+  }
+  ```
+  should replaced with
+  ```json
+  "dependencies": {
+    "@ud-viz/*": "file:../../../UD-Viz/packages/*"
```
Comment (EricBoix) [inline docs/static/Developers.md:102]: Are we sure to need that many `..` ? As newbie I was expecting `"file../UD-Viz/packages/*".

File: docs/static/Developers.md [hunk 2]
```diff
@@ -75,6 +75,57 @@ npm run dev-examples
...
+    //...
+    resolve: {
+      modules: ['../UD-Viz/node_modules', './node_modules'],
+    },
+  };
+  ```
+  It is important to add the '../UD-Viz/node_modules' in first position of the
+  array.
+
+Note that when you make a change in UD-Viz library watchers (nodemon) of Imuv will not notice it, you have to restart it yourself by typing "rs" in the watcher console.
```
Comment (EricBoix) [inline docs/static/Developers.md:127]: Remove the reference to Imuv and replace it with "your Application".
Clarify that this remark is only valid when your application uses nodemon ?
Something like (?)
When you application uses code watchers (e.g. `nodemon`) on automatic rebuild purposes, the code watchers will be blind to changes operated in the `../UD-Viz/` file hierarchy.

Comment (mathieuLivebardon) [reply docs/static/Developers.md:127]: It's true there are too many `..` 😶

File: docs/static/Developers.md (hunk 1 above)
Comment (mathieuLivebardon) [reply docs/static/Developers.md:102]: It's true there are too many .. 😶

**Review by EricBoix — APPROVED** [review]

**Pull request merged**